package.json

```diff
@@ -6,18 +6,27 @@
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "lint": "eslint './src/**/*.{js,jsx}'"
```

Comment:
this should just be `eslint`. AFAIK you already specified the files in eslint.config.js

Reply:
also, add one more script:

"lint: fix": "eslint --fix"